composer.json

```diff
@@ -30,6 +30,8 @@
         "sensio/framework-extra-bundle": "3.0.*",
         "stof/doctrine-extensions-bundle": "1.1.*",
         "symfony-cmf/routing-bundle": "1.2.*",
+        "symfony-cmf/core-bundle": "~1.1",
```

Comment:
Don't really want to include this, but is needed for the `SlugifierInterface`: https://github.com/symfony-cmf/CoreBundle/issues/165